packages/styles/src/react/private/useStyles.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+
+function useStyles<T extends CSSModuleClasses>(styles: T): T {
+  // @ts-expect-error: entries/fromEntries don't allow to specify keys type
+  return useMemo(
+    () =>
+      Object.freeze(
+        Object.fromEntries(
+          Object.entries(styles).map(([baseClassName, resultClassName]) => [
+            baseClassName,
+            `${baseClassName} ${resultClassName}`
```

Comment:
Question... why do we need to append the key to the class name? Is it only for React?

Reply:
Per convention we have in `component` package the classes look like: `webchat__block__element--modifier <generated-class-we-apply-styles-to>`.

In Web Chat we had to specify the `webchat__block__element--modifier` by hand. This reflects the convention and automates the part, so we in most cases we only need to: `className={classNames['something']}`.

We do so to provide clear naming for classes we are using, so they can be styled outside of Web Chat.